example-get-started/generate.sh

```diff
@@ -143,40 +143,55 @@ dvc push
 
 dvc run -n evaluate \
   -d src/evaluate.py -d model.pkl -d data/features \
-  -M evaluation/metrics.json \
-  -O evaluation/plots/metrics \
-  --plots-no-cache evaluation/plots/sklearn/roc.json \
-  --plots-no-cache evaluation/plots/sklearn/confusion_matrix.json \
-  --plots-no-cache evaluation/plots/prc.json \
-  --plots evaluation/plots/importance.png \
+  -o eval/importance.png -O eval/prc -O eval/live/plots \
+  -M eval/live/metrics.json \
   python src/evaluate.py model.pkl data/features
-dvc plots modify evaluation/plots/sklearn/roc.json -x fpr -y tpr
-dvc plots modify evaluation/plots/sklearn/confusion_matrix.json \
-    -x actual -y predicted -t confusion
-dvc plots modify evaluation/plots/prc.json -x recall \
-    -y precision
-git add .gitignore dvc.yaml dvc.lock evaluation
+git add .gitignore dvc.yaml dvc.lock eval
 tick
 git commit -m "Create evaluation stage"
-git tag -a "baseline-experiment" -m "Baseline experiment evaluation"
 git tag -a "8-evaluation" -m "Baseline evaluation stage created."
 dvc push
 
 
+echo "plots:
+  - eval/importance.png
+  - Precision-Recall:
+      x: recall
+      y:
+        eval/prc/train.json: precision
+        eval/prc/test.json: precision
+  - ROC:
+      x: fpr
+      y:
+        eval/live/plots/sklearn/roc/train.json: tpr
+        eval/live/plots/sklearn/roc/test.json: tpr
+  - Confusion-Matrix:
+      template: confusion
+      x: actual
+      y:
+        eval/live/plots/sklearn/cm/train.json: predicted
+        eval/live/plots/sklearn/cm/test.json: predicted" >> dvc.yaml
+git add dvc.yaml
+tick
+git commit -m "Configure plots"
+git tag -a "baseline-experiment" -m "Baseline experiment evaluation"
```

Comment:
feels like we can still do a single step for simplicity? evaluation w/o plots is not very meaningful, I guess.